nexus/pkg/server/filestream.go

```diff
@@ -225,6 +228,9 @@ func (fs *FileStream) doChunkProcess(inChan <-chan chunkData) {
 				}
 			}
 		}
+		if fs.stageExitChunk != nil {
```

Comment:
could we add a minor comment here that exit could come before other chunks and we store it until we exhaust all the other messages, ensuring it would be the last message?

Reply:
oh ignore saw your comment in streamFinish